nexus/completion.go

```diff
@@ -10,6 +10,9 @@ import (
 	"net/http"
 	"strconv"
 	"time"
+
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/types/known/timestamppb"
```

Comment:
Nexus doesn't depend on protobuf and shouldn't start depending on it now. You should copy over the code instead.